src/platform/cocoamain.mm

```diff
@@ -1218,6 +1218,227 @@ - (IBAction)preferences:(id)sender {
     [NSApp stop:nil];
 }
 
+#ifdef HAVE_SPACEWARE
```

Comment:
Yes, because it no longer should be linked to when using CFBundle, as I'm loading it dynamically...?